monocle/sanitized.py

```diff
@@ -66,6 +66,7 @@
     'LOGIN_TIMEOUT': Number,
     'MANAGER_ADDRESS': (str, tuple, list),
     'MAP_END': sequence,
+    'MAP_FILTER_IDS': (bool, list),
```

Comment:
This should be `sequence` (meaning `list` and `tuple`). `True` is not a valid option, and the provided default is [implicitly acceptable](https://github.com/Noctem/Monocle/blob/b140232e4342af62d7d06b34e9469846b5b40ce2/monocle/sanitized.py#L236). I personally prefer using tuples whenever it's immutable (as in this case).